googleanalytics/googleanalytics-connector/googleanalytics-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/googleanalytics-goals/component.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2005-2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+
+   WSO2 Inc. licenses this file to you under the Apache License,
+   Version 2.0 (the "License"); you may not use this file except
+   in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+-->
+<component name="googleanalytics-goals" type="synapse/template">
+    <subComponents>
+        <component name="listGoals">
+            <file>listGoals.xml</file>
+            <description>Retrieve a list of goals from GoogleAnalytics account.</description>
+        </component>
+        <component name="getGoals">
+            <file>getGoals.xml</file>
+            <description>Retrieve a goals from GoogleAnalytics account.</description>
+        </component>
+        <component name="insertGoals">
+            <file>insertGoals.xml</file>
+            <description>Retrieve a goals from GoogleAnalytics account.</description>
```

Comment:
Fixed in https://github.com/wso2-extensions/esb-connector-googleanalytics/pull/5